src/pydicom/data/data_manager.py

```diff
@@ -387,7 +398,17 @@ def get_testdata_files(pattern: str = "**/*") -> list[str]:
     -------
     list of str
         A list of absolute paths to matching files.
+
+    Raises
+    ______
+    ValueError
+        If `pattern` matches an absolute path.
     """
+    if os.path.isabs(pattern):
+        raise ValueError(
+            f"'get_testdata_files' does not support absolute paths, as it only works"
+            f" with internal pydicom test data."
+        )
```

Comment:
I thought in this case mentioning `dcmread` is not needed, as `get_testdata_files` is used nowhere in the documentation and the problem probably not a copy/paste error.

Reply:
Ah, yes, I didn't think that through correctly.